archway/chain.json

```diff
@@ -37,24 +37,23 @@
   },
   "codebase": {
     "git_repo": "https://github.com/archway-network/archway",
-    "recommended_version": "v7.0.1",
+    "recommended_version": "v8.0.0",
     "compatible_versions": [
-      "v7.0.0",
-      "v7.0.1"
+      "v8.0.0"
     ],
     "binaries": {
-      "linux/amd64": "https://github.com/archway-network/archway/releases/download/v7.0.1/archwayd_linux_amd64",
-      "linux/arm64": "https://github.com/archway-network/archway/releases/download/v7.0.1/archwayd_linux_arm64",
-      "darwin/amd64": "https://github.com/archway-network/archway/releases/download/v7.0.1/archwayd_darwin_amd64",
-      "darwin/arm64": "https://github.com/archway-network/archway/releases/download/v7.0.1/archwayd_darwin_arm64"
+      "linux/amd64": "https://github.com/archway-network/archway/releases/download/v8.0.0/archwayd_linux_amd64",
+      "linux/arm64": "https://github.com/archway-network/archway/releases/download/v8.0.0/archwayd_linux_arm64",
+      "darwin/amd64": "https://github.com/archway-network/archway/releases/download/v8.0.0/archwayd_darwin_amd64",
+      "darwin/arm64": "https://github.com/archway-network/archway/releases/download/v8.0.0/archwayd_darwin_arm64"
     },
     "cosmos_sdk_version": "v0.47.11",
     "ibc_go_version": "v7.4.0",
     "consensus": {
       "type": "cometbft",
       "version": "v0.37.5"
     },
-    "cosmwasm_version": "v0.45.0",
+    "cosmwasm_version": "archway-network/archway-wasmd v0.46.0-archway",
```

Comment:
Try to make this only `archway-wasmd v0.46.0-archway` and see if it passes the check